airflow-core/src/airflow/migrations/versions/0079_3_1_0_make_bundle_name_not_nullable.py

```diff
@@ -0,0 +1,103 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""
+Make bundle_name not nullable.
+
+Revision ID: fd0eb9b90774
+Revises: 09fa89ba1710
+Create Date: 2025-07-24 04:04:09.383329
+
+"""
+
+from __future__ import annotations
+
+from alembic import op
+from sqlalchemy.sql import text
+
+from airflow.migrations.db_types import StringID
+
+# revision identifiers, used by Alembic.
+revision = "fd0eb9b90774"
+down_revision = "09fa89ba1710"
+branch_labels = None
+depends_on = None
+airflow_version = "3.1.0"
+
+
+def upgrade():
+    """Make bundle_name not nullable."""
+    dialect_name = op.get_bind().dialect.name
+    if dialect_name == "postgresql":
+        op.execute("""
+                    INSERT INTO dag_bundle (name) VALUES
+                      ('example_dags'),
+                      ('dags-folder')
+                    ON CONFLICT (name) DO NOTHING;
+                    """)
+    if dialect_name == "mysql":
+        op.execute("""
+                    INSERT IGNORE INTO dag_bundle (name) VALUES
+                      ('example_dags'),
+                      ('dags-folder');
+                    """)
+    if dialect_name == "sqlite":
+        op.execute("""
+                    INSERT OR IGNORE INTO dag_bundle (name) VALUES
+                      ('example_dags'),
+                      ('dags-folder');
+                    """)
+
+    conn = op.get_bind()
+    with op.batch_alter_table("dag", schema=None) as batch_op:
+        conn.execute(
+            text(
```

Comment:
Why does this use `text()` but others don’t?

Reply:
This is optional in my case, because I am not binding any parameter in my queries. I will add text to all queries to align with other migration files.

Reply:
updated